requirements.txt

```diff
@@ -1,3 +1,4 @@
+alembic==1.8.0
```

Comment:
this also needs to be added to setup.py somewhere. i think this piece is pretty crucial and can be in minimal requirements.

Reply:
added to `setup.py` and pinned at 1.8.0